Portfolio/index.css

```diff
@@ -111,32 +116,174 @@ a {
     text-align: center;
 }
 
-.about{
+.about {
     display: flex;
     flex-direction: column;
     text-align: center;
     justify-content: center;
     padding: 2rem;
 }
-.about-title{
+
+.about-title {
     font-size: 2.5rem;
     letter-spacing: 0.03em;
     margin: 2rem;
 }
-.about-article{

```

Comment:
Consider uncommenting the height property for .get-in-touch to ensure all content is visible without overflow